xbmc/cores/VideoPlayer/Interface/Addon/InputStreamConstants.h

```diff
@@ -8,6 +8,35 @@
 
 #pragma once
 
-#define STREAM_PROPERTY_INPUTSTREAMCLASS  "inputstreamclass" /*!< @brief the name of the inputstream add-on that should be used by Kodi to play the stream denoted by STREAM_PROPERTY_STREAMURL. Leave blank to use Kodi's built-in playing capabilities or to allow ffmpeg to handle directly set to STREAM_PROPERTY_VALUE_INPUTSTREAMFFMPEG. */
-#define STREAM_PROPERTY_ISREALTIMESTREAM "isrealtimestream" /*!< @brief "true" to denote that the stream that should be played is a realtime stream. Any other value indicates that this is not a realtime stream.*/
-#define STREAM_PROPERTY_VALUE_INPUTSTREAMFFMPEG  "inputstream.ffmpeg" /*!< @brief special value for STREAM_PROPERTY_INPUTSTREAMCLASS to use ffmpeg to directly play a stream URL. */
+/*!
+ * @brief the name of the inputstream add-on that should be used by Kodi to
+ * play the stream denoted by STREAM_PROPERTY_STREAMURL. Leave blank to use
+ * Kodi's built-in playing capabilities or to allow ffmpeg to handle directly
+ * set to STREAM_PROPERTY_VALUE_INPUTSTREAMFFMPEG.
+ */
+#define STREAM_PROPERTY_INPUTSTREAMCLASS "inputstreamclass"
```

Comment:
> the name of the inputstream add-on

 1. name or id? At other places you wrote that this is the id
 2. if it is the name/id of the addon, this property should be named xyzID or yxzNAME to reflect what it actually represents.

Reply:
Is just an addon ID, that's just the text that has been moved from the one-liner.

I change it to match more the addons to `STREAM_PROPERTY_INPUTSTREAM_NAME` and the other then to `STREAM_PROPERTY_INPUTSTREAM_ID`. Only the text with "inputstreamclass" I must leave to allow compatibility.

Also about my added documentation must look, with my english them sounds sometimes stupid :wink:

Reply:
Becomes changed to:
```cpp
#define STREAM_PROPERTY_INPUTSTREAM_NAME "inputstreamclass"
```
Leaved as `inputstreamclass` to prevent API problems, think also somewhere used by python and not so easy to rename too.